File: Latihan5/C++/soal5.cpp
#include <iostream>

int main() {
    // Inisialisasi a
    int a = 10;

    // Loop i dari a hingga 1
    for (int i = a; i >= 1; i--) {
        // Loop n dari 1 hingga i
        for (int n = 1; n <= i; n++) {
            // Jika nilai Loop n tidak lebih besar dari a
            if (n <= a) {
                std::cout << i * n << " ";
            }
        }
        // Akhir loop n
        std::cout << std::endl;  // Cetak baris kosong
    }

    // Jika nilai loop i lebih besar dari a
    if (a < 1) {
        std::cout << "Program selesai" << std::endl;
    }

    return 0;
}

File: Latihan5/Python/soal5.py
# Inisialisasi a
a = 10

# Loop i dari a hingga 1
for i in range(a, 0, -1):
    # Loop n dari 1 hingga i
    for n in range(1, i + 1):
        # Jika nilai Loop n tidak lebih besar dari a
        if n <= a:
            print(i * n, end=" ")
    # Akhir loop n
    print()  # Cetak baris kosong

# Jika nilai loop i lebih besar dari a
if a < 1:
    print("Program selesai")
